docs/TerraToolkitUpgradeGuide-v5.0.0.md

```diff
@@ -0,0 +1,116 @@
+# Terra Toolkit Upgrade Guide v5.0.0
+This document will provide information on upgrading from terra-toolkit 4.x to 5.0.0.
+
+## Terra Scripts
+### Serve
+Serve is now a thin abstraction on webpack dev server and the commandline api is now identical. With this addition it now means that you have control over the dev server through options specified in your webpack config as well as through the cli.
+
+Why use serve instead of webpack-dev-server directly? Having the serve abstraction provides a hook for us to change the servers implementation in case webpack-dev-server no longer meets our needs.
+
+Our default webpack config has been updated with the following defaults:
+
+```javascript
+module.exports = {
+  devServer: {
+    // We set the host this way to allow it to be served from docker containers.
+    host: '0.0.0.0',
+    // These options cut down on noise in the webpack build output.
+    stats: {
+      colors: true,
+      children: false,
+    },
+  },
+};
+```
+
+Serve is no longer available as a javascript function. If this is needed use webpack-dev-server directly.
```

Comment:
lets add an example of the expected code changes & link webpack-dev-server 

Also, would you expect consumers to use the webpack-cli script to run webpack-dev-server?https://github.com/webpack/webpack-cli/blob/master/packages/serve/README.md#webpack-cli-serve

Reply:
I can link webpack-dev-server, but I don't want to add an example. I don't think people are doing this today and using webpack dev server this way can get kind of hard to cover all use-cases.

No i don't expect people to use the webpack-cli serve command. I expect them to use tt-serve.